src/FixedDataTable.js

```diff
@@ -1196,8 +1196,8 @@ class FixedDataTable extends React.Component {
     }
 
     // only call onScrollStart if scrolling wasn't on previously
-    if (!this.props.scrolling && onScrollStart) {
-      onScrollStart(oldScrollX, oldScrollY, oldFirstRowIndex, oldEndRowIndex)
+    if (!prevProps.scrolling && this.props.scrolling && onScrollStart) {
```

Comment:
Let's destructure `scrolling` from `prevProps` and `this.props` above.
Then `scrolling` will refer to the new value, and `oldScrolling` will refer to the previous value.